apps/web/src/app/terms-of-use/page.tsx

```diff
@@ -0,0 +1,970 @@
+"use client";
+import React from "react";
+import Showcase from "@/common/Showcase";
+import { Container } from "@/configs/mui";
+import { TermsOfUseWrapper } from "./styles";
+
+type Props = {};
+
+function page({}: Props) {
```

Comment:
these info should come from the backed correct? If yes we should have them as object/list on top with the proper TODO